src/python_pachyderm/mixin/debug.py

```diff
@@ -1,13 +1,22 @@
 from typing import Iterator
-from python_pachyderm.service import Service, debug_proto
+
+import grpc
 from google.protobuf import duration_pb2
 
+from python_pachyderm.proto.v2.debug import debug_pb2, debug_pb2_grpc
+
 
 class DebugMixin:
     """A mixin for debug-related functionality."""
 
+    _channel: grpc.Channel
+
+    def __init__(self):
+        self.__stub = debug_pb2_grpc.DebugStub(self._channel)
```

Comment:
It's funny to me that proto decides to spit out `DebugStub` rather than `ApiStub` here.